contracts/mixnet/src/storage.rs

```diff
@@ -193,11 +195,14 @@ pub(crate) fn increase_mix_delegated_stakes(
         );
 
         // and for each of them increase the stake proportionally to the reward
+        // if they have delegated for long enough
```

Comment:
I'd add a more verbose explanation for `long enough`, something along the lines, "At least N blocks have been created since they've delegated"